provider/application-standard/src/main/java/com/speedment/jpastreamer/application/standard/internal/StandardStreamSupplier.java

```diff
@@ -63,8 +67,17 @@ public Stream<T> stream() {
     }
 
     @Override
-    public void close() {
-        //System.out.println("Closing Streamer<" + entityClass.getSimpleName() + ">");
-        renderer.close();
+    public void close() throws UnsupportedOperationException {
+        if (this.closeEntityManager) {
+            renderer.close();
+        } else {
+            System.out.println("JPAStreamer does not close Entity Managers obtained by a given Supplier<EntityManager>.");
```

Comment:
Maybe we should only print this message once. So we could have a static field of type `AtomicBoolean` that controls this.

Reply:
I think we need to discuss this briefly. This slight complexity arose due to compatibility with Quarkus.